modules/synthetic-protocol/src/lib.rs

```diff
@@ -74,10 +74,10 @@ decl_module! {
 			pool_id: LiquidityPoolId,
 			currency_id: CurrencyId,
 			#[compact] collateral_amount: Balance,
-			max_slippage: Permill
+			max_price: Price,
```

Comment:
Does the price type need to add `#[compact]` ?

Reply:
I think `Price` doesn't support compact no?

Reply:
I think we need it. Not sure if it will work but it is possible to implement `HasCompact` for `Price`

Can you try add to see if it compiles, if not, leave it is and I will make an issue in ORML and we can add it later.

Reply:
Compile failed.
required because of the requirements on the impl of `_::_parity_scale_codec::HasCompact` for `orml_utilities::fixed_u128::FixedU128`

Reply:
https://github.com/laminar-protocol/open-runtime-module-library/issues/134